src/openst.js

```diff
@@ -21,8 +21,8 @@ class OpenST {
     const tokenRulesTxOptions = this.auxiliary.txOptions;
     const tokenRules = await TokenRules.deploy(
       this.auxiliary.web3,
-      auxiliaryOrganization,
-      auxiliaryEIP20Token,
+      auxiliaryOrganization.address,
+      auxiliaryEIP20Token.address,
```

Comment:
This method is also called from openst.js executable.
The arguments passed are address and not account object. Since the method is called from multiple places, I will suggest to keep type of arguments consistent.
I am not sure how we can pass account object from command line 😕 
I will suggest to keep the argument types as address.

Reply:
Ah, ok! I thought `auxiliaryOrganization`, `auxiliaryUtilityToken`, referenced objects in `setupOpenst` because that's the case in `_deployUtilityToken`. JLP!

But also, based on your other comment, I realize now this was a result of more changes than strictly necessary. Reverted!